libraries/Microsoft.Bot.Builder.Dialogs.Declarative/Converters/InterfaceConverter.cs

```diff
@@ -24,6 +24,9 @@ public class InterfaceConverter<T> : JsonConverter, IObservableConverter, IObser
         private readonly List<IJsonLoadObserver> observers = new List<IJsonLoadObserver>();
         private readonly SourceContext sourceContext;
 
+        private Dictionary<string, JToken> cachedRefJsons = new Dictionary<string, JToken>();
```

Comment:
Use `ConcurrentDictionary` to ensure the thread safety

Reply:
fixed.